crm_claim/migrations/11.0.2.0.0/pre-migration.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2018 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, SUPERUSER_ID
+
+
+def migrate(cr, version):
+    if not version:
+        return
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    activity_type = env.ref('mail.mail_activity_data_todo')
+    crm_claim_model = env.ref('crm_claim.model_crm_claim')
+    cr.execute("""
+        SELECT
```

Comment:
You can make the insert from a select query and make this faster

Reply:
Change done